app/models/tournament.rb

```diff
@@ -68,6 +68,16 @@ def locked_deck_players
     players.active.where('decks_locked IS TRUE')
   end
 
+  def decks_status_description
+    if all_players_decks_unlocked?
+      'all unlocked'
+    elsif any_player_decks_unlocked?
+      'partially unlocked'
```

Comment:
how about 'some unlocked'?

Reply:
Some suggests more than one. Partially seems more correct to me, what do you think?

Reply:
i don't have strong feelings about this.